db/schema.rb

```diff
@@ -15,7 +15,7 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "bookmarks", force: :cascade do |t|
+  create_table "bookmarks", id: :serial, force: :cascade do |t|
```

Comment:
@nomadicoder We are trying to investigate why exports are not working on prod.  We were wondering what this change was for.

Reply:
That was added during a `rails db:migrate`. I presume it's something added by Rails.

Reply:
Could have also been added if we performed a `rails db:setup.`